kafka/resource_kafka_acl.go

```diff
@@ -60,15 +61,22 @@ func kafkaACLResource() *schema.Resource {
 				ForceNew: true,
 			},
 		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Read:   schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
 	}
 }
 
 func aclCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	c := meta.(*LazyClient)
+	timeout := d.Timeout(schema.TimeoutCreate) - time.Minute
```

Comment:
Subtract time.Minute as suggested in the Terraform Plugin SDKv2 docs
https://www.terraform.io/plugin/sdkv2/resources/retries-and-customizable-timeouts#retry